<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textEdit">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit1">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit2">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit3">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit4">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit5">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="3">
         <widget class="QPushButton" name="pushButtonMultiply">
          <property name="text">
           <string>*</string>
          </property>
          <property name="shortcut">
           <string>*</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton9">
          <property name="text">
           <string>9</string>
          </property>
          <property name="shortcut">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="pushButton4">
          <property name="text">
           <string>4</string>
          </property>
          <property name="shortcut">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButtonC">
          <property name="text">
           <string>C</string>
          </property>
          <property name="shortcut">
           <string>C</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton8">
          <property name="text">
           <string>8</string>
          </property>
          <property name="shortcut">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="pushButtonPoint">
          <property name="text">
           <string>.</string>
          </property>
          <property name="shortcut">
           <string>.</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonSin">
          <property name="text">
           <string>sin</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="pushButtonAdd">
          <property name="text">
           <string>+</string>
          </property>
          <property name="shortcut">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButtonPower">
          <property name="text">
           <string>^</string>
          </property>
          <property name="shortcut">
           <string>^</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QPushButton" name="pushButtonDivide">
          <property name="text">
           <string>/</string>
          </property>
          <property name="shortcut">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="pushButton2">
          <property name="text">
           <string>2</string>
          </property>
          <property name="shortcut">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonRoot">
          <property name="text">
           <string>r</string>
          </property>
          <property name="shortcut">
           <string>R</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton7">
          <property name="text">
           <string>7</string>
          </property>
          <property name="shortcut">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pushButton5">
          <property name="text">
           <string>5</string>
          </property>
          <property name="shortcut">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="pushButton3">
          <property name="text">
           <string>3</string>
          </property>
          <property name="shortcut">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="pushButtonBack">
          <property name="text">
           <string>‚Üê</string>
          </property>
          <property name="shortcut">
           <string>Backspace</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="pushButton1">
          <property name="text">
           <string>1</string>
          </property>
          <property name="shortcut">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButtonCos">
          <property name="text">
           <string>cos</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton0">
          <property name="text">
           <string>0</string>
          </property>
          <property name="shortcut">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButtonTan">
          <property name="text">
           <string>tan</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QPushButton" name="pushButtonEqual">
          <property name="text">
           <string>=</string>
          </property>
          <property name="shortcut">
           <string>Enter</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="pushButton6">
          <property name="text">
           <string>6</string>
          </property>
          <property name="shortcut">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="pushButtonSubstract">
          <property name="text">
           <string>-</string>
          </property>
          <property name="shortcut">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>607</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionContact"/>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionContact">
   <property name="text">
    <string>Contact</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
